CHANGELOG.md

```diff
@@ -10,6 +10,14 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
   and `Environment`.
 - Additional transport headers were added to `transport.Response`: `ID`,
   `Host`, `Environment` and `Service`.
+- Added `peer/peerlistv2` which differs from the original `peer/peerlist` by
```

Comment:
From https://research.swtch.com/vgo-module, seems like you'd want to do `peer/peerlist/v2` which would continue to use the package name peerlist.